src/test/java/wooteco/subway/admin/acceptance/WholeLineStationAcceptanceTest.java

```diff
@@ -0,0 +1,52 @@
+package wooteco.subway.admin.acceptance;
+
+import static org.assertj.core.api.Assertions.*;
+
+import java.util.List;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import wooteco.subway.admin.dto.LineDetailResponse;
+import wooteco.subway.admin.dto.LineResponse;
+import wooteco.subway.admin.dto.StationResponse;
+
+public class WholeLineStationAcceptanceTest extends AcceptanceTest {
+
+    @DisplayName("전체 노선별 지하철 역 전체 조회")
+    @Test
+    void showWholeLineStation() {
+        LineResponse lineResponse1 = createLine("브라운선");
```

Comment:
```suggestion
        LineResponse 브라운선 = createLine("브라운선");
```
테스트 코드도 변수명을 읽기 좋게 만들어야 코드 파악하기 용이해져요